server/models/userModel.js

```diff
@@ -16,12 +16,12 @@ const user = {
     );
   },
   uploadPDF: async (url, fileUse, id) => {
-    return pool.query(`UPDATE Users SET ${fileUse}_pdf_url = $1
+    return pool.query(`UPDATE Users SET ${fileUse}_url = $1
       WHERE id=$2;`,
     [url, id])
   },
   getPDF: (fileUse, id) => {
-    return pool.query(`SELECT ${fileUse}_pdf_url FROM Users WHERE id=$1`, [id]);
+    return pool.query(`SELECT ${fileUse}_url FROM Users WHERE id=$1`, [id]);
```

Comment:
This change will require any other controller using getPDF to be refactored to match. I can take care of this, but it is an important consideration.

Reply:
I changed everything as required and it's working for both PDF and avatar image uploads now